### PR TITLE
Make ExpandVolumeDevice() idempotent if existing volume capacity meets the requested size.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by humblec** [comment]
I am working on another approach , till then please hold your review .

File: pkg/volume/glusterfs/glusterfs.go [hunk 1]
```diff
@@ -1132,6 +1132,22 @@ func (plugin *glusterfsPlugin) ExpandVolumeDevice(spec *volume.Spec, newSize res
 	expansionSize := (newSize.Value() - oldSize.Value())
 	expansionSizeGiB := int(volume.RoundUpSize(expansionSize, volume.GIB))
 
+	// Find out requested Size
+	requestBytes := newSize.Value()
+	requestGiB := volume.RoundUpSize(requestBytes, 1024*1024*1024)
```
[the remark this answers is not in the document]
Comment (humblec) [reply pkg/volume/glusterfs/glusterfs.go]: @gnufied done. Thanks!

**Comment by gnufied** [comment]
Fixes https://github.com/kubernetes/kubernetes/issues/58615

**Comment by gnufied** [comment]
@humblec can you link github issue in PR?

**Pull request merged**